src/PKSim.Assets.Images/PKSim.Assets.Images.csproj

```diff
@@ -26,8 +26,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="OSPSuite.Assets" Version="11.0.72" />
-    <PackageReference Include="OSPSuite.Assets.Images" Version="11.0.72" />
+	  <PackageReference Include="OSPSuite.Assets" Version="11.0.77" />
```

Comment:
there is some weird indent in every file @georgeDaskalakis

Reply:
correction in some files. Can you clean that up?

Reply:
done